personal_assistant/pa_tag/urls.py

```diff
@@ -0,0 +1,11 @@
+from django.urls import path
+
+from .views import main, add_tag, delete
+
+app_name = 'pa_tags'
```

Comment:
Краще нехай називається як і апака - в однині:
```suggestion
app_name = 'pa_tag'
```